### ISSUE TITLE
Why is the genome not circularized ?

### ISSUE THREAD

**Issue by Chloisf** [ndierckx/NOVOPlasty#142]
Dear Dr. Dierckxsens,

I have been using NOVOPlasty to assemble mt genomes of zoantharians using short reads (150bp) data of whole genome and a reference mtgenome.
For the two species I tried, I obtained an assembly in one contig, of around 21000bp which is what to expect from the litterature of my organism. However, the assembly does not seem to be circularized as the output is Contigs_1_project.fasta and not Circularized_assembly_project.fasta 
Since the genome seems to have expected size I am wondering why it could not be circularized. Do you have advice about how to circularize it ?

Thank you for NOVOPlasty and your attention!
Chloé

**Comment by ndierckx** [comment]
Hi,

I can't know that without more info. run with extended log and send that file

**Comment by Chloisf** [comment]
Hello!

I'm sorry for my late answer, it took me a while to be able to rerun.
Here is the extended log.
Thank you !
[log_extended_mtPheliodiscus3.txt](https://github.com/ndierckx/NOVOPlasty/files/4834942/log_extended_mtPheliodiscus3.txt)

**Comment by ndierckx** [comment]
Hi,

Seems there is a gap in coverage, so don't think it can be circularized completely. It's probably because of low complexity region of Cs. But I guess the gap is very small or maybe nothing because it seems both ends overlap a bit

**Comment by Chloisf** [comment]
Hello,

Thank you for taking the time to check my log!
Ok, so if both ends overlap even just a bit I assume the genome is still complete ? And I should be able to extract some genes to use them somehow.
Thank you for the explanations!

**Comment by ndierckx** [comment]
Yeah sure, if you miss a few basepares, it is not a problem, probably no gene is affected, many published mitogenomes have bigger gaps
YOu can check the overlap and clip a few basepares manually so it matches
